### DISCUSSION TITLE
Artemis Version for Samsung Tizen TVs

### DISCUSSION THREAD

**Discussion by marcelmira** [ClassicOldSong/Apollo#148] [Ideas]
Today there is no official version of moonlight for Tizen, just some ports that don't work very well, in addition to being complicated to install on the TV manually using docker, Tizen Studio, etc. Your work is great with Apollo/Artemis, and if you have the time to make this version for Tizen, I know that soooo many Samsung TV users would thank you forever. <3

**Comment by ClassicOldSong** [comment]
Thanks for your appreciation, but unfortunately I don't have a Samsung TV and I don't plan to get one as I'm pretty happy with my LG TV right now. Also I'm quite busy recently just to make a living.

**Comment by marcelmira** [comment]
What a shame :(. But thanks for the quick response and keep up the good work.

**Comment by dos-ise** [comment]
Would also love to have this.

I am currently using this client on my samsung tizen tv.

https://github.com/OneLiberty/moonlight-chrome-tizen

**Reply by ClassicOldSong** [reply]
I don't have that much time and there's currently no point to make Artemis for TV, nothing really meaningful can be provided for a fork.

**Reply by cidmartins** [reply]
> Would also love to have this.
> 
> I am currently using this client on my samsung tizen tv.
> 
> https://github.com/OneLiberty/moonlight-chrome-tizen

so you have managed to pair apollo with this moonlight fork ? for some reason , i get an error when pairing .

with sunshine it works though. 

I'll try again later.
